### ISSUE TITLE
Django 1.9b1: cannot import name 'ForeignRelatedObjectsDescriptor'

### ISSUE THREAD

**Issue by jbsag** [vandersonmota/model_mommy#259]
Django version: 1.9b1
```
./manage.py shell
 Python 3.5.0 (default, Sep 23 2015, 04:42:00)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>from model_mommy import mommy
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/anon/.virtualenvs/sl2/lib/python3.5/site-packages/model_mommy/mommy.py", line 24, in <module>
    from django.db.models.fields.related import ForeignRelatedObjectsDescriptor
ImportError: cannot import name 'ForeignRelatedObjectsDescriptor'
```

It looks like ``ForeignRelatedObjectsDescriptor`` got renamed to ``ReverseManyToOneDescriptor`` in Django 1.9.

Commit from Django: https://github.com/django/django/commit/e542e81b39e8610b70cf1d68f95ece0279028765#diff-3010fc5a498b7171c342520f34507968L435

I'd be happy to try to put together a pull request if I can figure out how to fix this without breaking anything, since Django 1.9 is set to release soon in December.

**Comment by hyzhak** [comment]
:+1:

**Comment by vandersonmota** [comment]
you could run tox if you're going to work on this @jbsag

**Comment by pmdarrow** [comment]
@jbsag are you taking this on? If not, I can pull together a PR.

**Comment by pmdarrow** [comment]
Nevermind, looks like @barbuza has already fix this in https://github.com/vandersonmota/model_mommy/pull/260/files.

**Comment by vandersonmota** [comment]
People, i'll review the remaining PRs/issues this weekend.

[]'s

**Comment by vandersonmota** [comment]
merged @barbuza fix into development branch: https://github.com/vandersonmota/model_mommy/tree/development
You may want to use this branch while we fix the remaining tests (django 1.9)
